src/libraries/System.Security.Cryptography.Encoding/tests/Cbor.Tests/CborWriterTests.cs

```diff
@@ -43,5 +45,70 @@ public static void BytesWritten_SingleValue_ShouldReturnBytesWritten()
             writer.WriteTextString("test");
             Assert.Equal(5, writer.BytesWritten);
         }
+
+        [Theory]
+        [MemberData(nameof(EncodedValueInputs))]
+        public static void WriteEncodedValue_RootValue_HappyPath(string hexEncodedValue)
+        {
+            byte[] encodedValue = hexEncodedValue.HexToByteArray();
+
+            using var writer = new CborWriter();
+            writer.WriteEncodedValue(encodedValue);
+
+            string hexResult = writer.ToArray().ByteArrayToHex();
+            Assert.Equal(hexEncodedValue, hexResult.ToLower());
+        }
+
+        [Theory]
+        [MemberData(nameof(EncodedValueInputs))]
+        public static void WriteEncodedValue_NestedValue_HappyPath(string hexEncodedValue)
+        {
+            byte[] encodedValue = hexEncodedValue.HexToByteArray();
+
+            using var writer = new CborWriter();
+            writer.WriteStartArray(3);
+            writer.WriteInt64(1);
+            writer.WriteEncodedValue(encodedValue);
+            writer.WriteTextString("");
+            writer.WriteEndArray();
+
+            string hexResult = writer.ToArray().ByteArrayToHex();
+            Assert.Equal("8301" + hexEncodedValue + "60", hexResult.ToLower());
+        }
+
+        [Fact]
+        public static void WriteEncodedValue_BadIndefiniteLengthStringValue_ShouldThrowInvalidOperationException()
```

Comment:
From a black box/scenario (vs ccov) perspective, it's probably good to have tests that show writing into each of the nesting constructs works.

* definite array
* indefinite array
* definite map, key
* definite map, value
* indefinite map, key
* indefinite map, value
* indefinite text string
* indefinite byte string

I think right now there's only one for definite array.  Tags should probably be sprinkled throughout, too.

Reply:
Done.